lib/medium-admin.php

```diff
@@ -136,10 +136,16 @@ public static function add_meta_boxes_post($post) {
   public static function save_post($post_id, $post) {
     if (defined('DOING_AUTOSAVE') && DOING_AUTOSAVE) return;
 
+    $allowed_post_types = (array) apply_filters('medium_allowed_post_types', array('post'));
```

Comment:
@majelbstoat Yes and that's a problem, don't you think? Posts, Pages, Nav Menu Items, Attachments...these are all post types in WordPress that I don't suspect should be getting sent to Medium with every save.